common/scala/src/main/scala/whisk/core/database/CouchDbRestStore.scala

```diff
@@ -163,8 +158,6 @@ class CouchDbRestStore[DocumentAbstraction <: DocumentSerializer](
                     // FIXME remove mutability from appropriate classes now that it is no longer required by GSON.
                     deserialized.asInstanceOf[WhiskDocument].revision(DocRevision(responseRev))
 
-                    deserialized
```

Comment:
It is, but it's now IMHO less readable than before -- logically, first line applies a side effect, second line returns.

Case in point: you couldn't tell.